proto/dydxprotocol/clob/query.proto

```diff
@@ -178,17 +178,18 @@ message StreamOrderbookUpdatesResponse {
 // GRPC stream.
 message StreamUpdate {
   // Contains one of an StreamOrderbookUpdate,
-  // StreamOrderbookFill.
+  // StreamOrderbookFill, StreamTakerOrderStatus.
   oneof update_message {
     StreamOrderbookUpdate orderbook_update = 1;
     StreamOrderbookFill order_fill = 2;
+    StreamTakerOrderStatus taker_order_status = 3;
   }
 
   // Block height of the update.
-  uint32 block_height = 3;
+  uint32 block_height = 4;
```

Comment:
just to make sure - is this schema backwards compatible?

Reply:
Since this is emitted in fns only, no need for it to be backwards compatible?